build/gulp/tasks/stats.ts

```diff
@@ -114,6 +123,7 @@ task('stats:build:bundle', async () => {
     .value()
 
   updateStatsFile(paths.docsSrc('bundleStats.json'), results)
```

Comment:
Later, we will probably get rid of the local stats file in favor of the external DB